src/cloud-api-adaptor/test/e2e/assessment_helpers.go

```diff
@@ -13,7 +13,7 @@ import (
 	"time"
 
 	log "github.com/sirupsen/logrus"
-	"gopkg.in/yaml.v2"
+	yaml "gopkg.in/yaml.v2"
```

Comment:
I think you did, but double-checking: compiled e2e tests for each provider? (this is something good to have on CI too).

Reply:
I ran local linting, but haven't compiled them for each provider, so I can try that out